CPs/cp_nn6.adoc

```diff
@@ -835,7 +835,7 @@ Table 3.52.2-1: Actor Roles
 |Reports confirmed association events to consumers. The manager must provide a HMI to verify association and disassociation assertions from a reporter if required, and once verified it persists the record and reports it to any consumers configured to receive the events in real-time. The manager should support filtering of messages, and may support dynamic filtering requested by the consumer. The manager must send current associations for all devices that the consumer is configured to receive reports for immediately after a connection is established.
 
 |Device-Patient Association Consumer
-|The receiver of the verified and final association report.  The consumer initially receives current association status followed by updates in real-time on a connection established by the Manager. It may optionally send a filter dynamically in the form a HL7 query or that filter may be pre-configured. When a association report is successfully received, a commit-level accept acknowledgement must be returned the the Manager.
+|The receiver of the verified and final association report. The Consumer may optionally initiate a subscription by sending a message with filtering criteria, if any, to the Manager in the form a HL7 query. The subscription and filter may also be pre-configured on the Manager. The Consumer initially receives current association status followed by updates in real-time on a connection established by the Manager.  When a association report is successfully received, a commit-level accept acknowledgement must be returned to the Manager.
 
```

Comment:
"a association report" -> "an association report.."

Reply:
Made this change.